README.md

```diff
@@ -54,6 +52,68 @@ VitePWA({
 })
 ```
 
+### Prompt for new content 
+
+![](https://user-images.githubusercontent.com/11247099/111190584-330cf880-85f2-11eb-8dad-20ddb84456cf.png)
+
+```ts
+// main.ts
+import { registerSW } from '@virtual/pwa-register'
```

Comment:
should be (you renamed the prefix and include the target):
```typescript
import { useRegisterSW } from 'virtual:pwa-register/vue'

const updateSW = registerSW({
```

Reply:
this plugin is framework agnostic, and so, only updated the import for the virtual module: below we have a link to a `vue 3` sample